webcomponents/tool/src/main/frontend/sakai-editor.js

```diff
@@ -0,0 +1,42 @@
+import {SakaiElement} from "./sakai-element.js";
+import {html} from "./assets/lit-element/lit-element.js";
+
+class SakaiEditor extends SakaiElement {
+    static get properties() {
+        return {
+            textAreaId: { type: String },
+            text: { type: String }
+        };
+    }
+
+    constructor() {
+        super();
+        console.log("Sakai Editor constructor");
+        this.textAreaId = 'editor';
+        this.text = 'Add your content here';
+    }
+
+    render() {
+        console.log("Sakai Editor render");
+
+        return html`
+            <textarea id="${this.textAreaId}">${this.text}</textarea>
+        `;
+    }
+
+    firstUpdated(changedProperties) {
+        console.log("Sakai Editor firstUpdated");
+        const textArea = this.querySelector(`#${this.textAreaId}`);
+        textArea.focus();
```

Comment:
You should probably focus the editor using the api, rather than the text area that will be replaced (it might not even be  a text area.